.circleci/config.yml

```diff
@@ -17,6 +17,9 @@ jobs:
       - checkout
       - attach_workspace:
           at: .
+      - run: 'sudo apt-get update'
+      - run: 'sudo apt-get install cppcheck'
+      - run: 'cppcheck *.* --language=c++ --enable=warning --error-exitcode=1'
```

Comment:
ez nyugodtan mehet egy masik job-ba. meg igazabol az sca elobb lemehet mint a buildeles is akar.

Reply:
@hegyhati a kommentednek megfelelően javítottam.